hawkular-alerts-rest/src/main/java/org/hawkular/alerts/rest/TriggersHandler.java

```diff
@@ -514,45 +839,84 @@ public Response createCondition(
             final String triggerId,
             @ApiParam(value = "Json representation of a condition. For examples of Condition types, See "
                     + "https://github.com/hawkular/hawkular-alerts/blob/master/hawkular-alerts-rest-tests/"
-                    + "src/test/groovy/org/hawkular/alerts/rest/ConditionsITest.groovy")
+                    + "src/test/groovy/org/hawkular/alerts/rest/ConditionsITest.groovy")//
             String jsonCondition) {
```

Comment:
I think this is something I can review in the future. Not part of this PR itself but I guess that with an specific annotation on abstract Condition class, the work of deserialize in inner specific conditions can be done in a transparent way. Instead to do it explicitly here.
It works as it is now, but just a point to improve in the future.